app/src/main/res/values/themes.xml

```diff
@@ -1,4 +1,10 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- Splash screen theme -->
+    <style name="Theme.MySplash" parent="Theme.SplashScreen">
```

Comment:
rename theme Splash theme to Theme.BC_VAX.Splash

Reply:
done